financial_time_series/README.md

```diff
@@ -0,0 +1,285 @@
+Our cluster is now up and running and properly set up in order to install kubeflow.
+```
+export KUBEFLOW_VERSION=0.2.2
+curl https://raw.githubusercontent.com/kubeflow/kubeflow/v${KUBEFLOW_VERSION}/scripts/deploy.sh | bash
+```
+Note that it requires only a single command to deploy Kubeflow to an existing cluster.
+Once the script is finished, you should two new folders in your directory
+```
+$ tree
+.
+├── kubeflo_ks_app
+└── kubeflow_repo
+```
+Next, we can easily verify the status of the pods by running ```kubectl get pods```:
+```
+NAME                                        READY     STATUS
+ambassador-788655d76f-8fkpv                 2/2       Running
+ambassador-788655d76f-fvjld                 2/2       Running
+ambassador-788655d76f-t4xqt                 2/2       Running
+centraldashboard-6665fc46cb-jrwvf           1/1       Running
+spartakus-volunteer-9c546f4db-5pztt         1/1       Running
+tf-hub-0                                    1/1       Running
+tf-job-dashboard-644865ddff-fbwnw           1/1       Running
+tf-job-operator-v1alpha2-75bcb7f5f7-fgf9h   1/1       Running
+```
+
+Note that the corresponding services were also generated by checking the output of ```kubectl get svc``` .
+
+### Exploration via tf-hub
+The tf-hub component of Kubeflow allows us to leverage Jupyter Notebooks to investigate the data and start building a feasible machine learning model for our problem statement.
+In order to access this component, we will set up port-forwarding between the tf-hub pod and our local machine.
+```
+POD=`kubectl get pods --selector=app=tf-hub | awk '{print $1}' | tail -1`
+kubectl port-forward $POD 8000:8000 2>&1 >/dev/null &
+```
+You should now be able to access the JupyterHub via ```localhost:8000```.
+After filling in a username and password you are prompted to select parameters to spawn a Jupyter Notebook.
+In this case, we will just set the ```image``` to ```gcr.io/kubeflow-images-public/tensorflow-1.8.0-notebook-cpu:v0.2.1``` and hit spawn.
+
+Once the Jupyter Notebook instance is ready, we will launch a terminal to install the required packages that our code uses.
+In order to launch a terminal, click 'new' > 'terminal' and subsequently install the required packages.
+```
+pip3 install google-cloud-bigquery==1.5.0 --user
+```
+
+Our Jupyter Notebook instance should be ready to run the code from the slightly adjusted notebook which is available on the github repository ```Machine Learning with Financial Time Series Data.ipynb```.
+You can simply upload the notebook and walk through it step by step to better understand the problem and suggested solution(s).
+In this blogpost, the goal is not focus on the notebook itself but rather on how this notebook is being translated in more scalable training jobs and later on serving.
```

Comment:
In this example? Perhaps you're posting this as a blog post elsewhere which is great!